discover/portal/articles/130-forms/03-form-rules/03-enable-rules.markdown

```diff
@@ -0,0 +1,39 @@
+# Action: Enable and Disable
+
+Use an enable/disable rule to make a field editable based on one or more conditions.
+
++$$$
+
+**Example:** Part of the race registration fee pays for dog food. You don't have
+to feed your chicken team with the provided dog food though. There's a single
+selection field that asks *Would you like use the provided dog food?*. If you
+check the box, you can select how much food, in US pounds, you'll need for your
```

Comment:
Good catch, let's change it to say:
"There's a single
selection field that asks *Would you like use the provided dog food?*. If you
select *Yes*, you can select how much food, in US pounds, you'll need for your
team throughout the race. Since you're racing chickens, you'll select *No*, and
the *Amount (US lb.)* field will be grayed out and disabled."